### ISSUE TITLE
Remove the index.js file in the native app

### ISSUE THREAD

**Issue by amerllica** [butadpj/turborepo-with-expo-router#8]
Remove the `index.js` file, because it contains only an entry file which could be in the `package.json` file.

**Comment by butadpj** [comment]
Yes, that's what I thought as well. But, for some reason, when you start building your app with EAS Build, it doesn't work. You can try it yourself and let me know if it work so I can merge the PR

**Comment by amerllica** [comment]
@butadpj, Sorry but what is the EAS build? Have you got any doc links?

**Comment by butadpj** [comment]
@amerllica, _[EAS Build](https://docs.expo.dev/build/introduction/) is a hosted service for building app binaries for your Expo and React Native projects._ Instead of us building our app's binaries (.apk) locally, you do it on the cloud with EAS Build. The advatange of building with EAS Build is it makes distributing our apps internally (for testing purposes) much easier. It also helps submitting our apps to PlayStore/AppStore much easier
